clayui.com/src/components/IconSearch/IconSearch.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React, {useEffect, useState} from 'react';
+
+import icons from './icons.json';
+
+const spritemap = '/images/icons/icons.svg';
+
+export default () => {
+	const [searchQuery, setSearchQuery] = useState('');
+	const [filteredIconNames, setFilteredIconNames] = useState([]);
+
+	const handleIconSearch = event => {
```

Comment:
No need to declare this separately from the `onChange` prop below. We can inline this function directly to the jsx.